infra.bs

```diff
@@ -107,12 +107,61 @@ leading spaces" or "return false") are to be interpreted with the meaning of the
 manner, so long as the end result is equivalent. (In particular, the algorithms are intended to be
 easy to follow, and not intended to be performant.)
 
+<h4 id=algorithm-control-flow>Control flow</h4>
+
 <p>The control flow of algorithms is such that a requirement to "return" or "throw" terminates the
 algorithm the statement was in. "Return" will hand the given value, if any, to its caller. "Throw"
 will make the caller automatically rethrow the given value, if any, and thereby terminate the
 caller's algorithm. Using prose the caller has the ability to "catch" the exception and perform
 another action.
 
+<p>An iteration's flow can be controlled via requirements to
+<dfn export for=iteration>continue</dfn> or <dfn export for=iteration>break</dfn>.
```

Comment:
@domenic if we export these, why not export return and throw? Is the plan to do that later once we figure things out with IDL?